migrations/versions/0343_org_billing_details.py

```diff
@@ -0,0 +1,65 @@
+"""
+
+Revision ID: 0343_org_billing_details
+Revises: 0342_service_broadcast_settings
+Create Date: 2021-02-01 14:40:14.809632
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0343_org_billing_details'
+down_revision = '0342_service_broadcast_settings'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('organisation', sa.Column('billing_contact_email_addresses', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('billing_contact_names', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('billing_reference', sa.String(length=255), nullable=True))
+    op.add_column('organisation', sa.Column('notes', sa.Text(), nullable=True))
+    op.add_column('organisation', sa.Column('purchase_order_number', sa.String(length=255), nullable=True))
+    op.alter_column('services', 'billing_contact_email_addresses',
```

Comment:
Maybe I'm looking at the wrong thing here, but it looks to me like the `billing_contact_email_addresses` and `billing_contact_names` columns on `services` and `services_history` already had the type of `Text`?

That's from looking at the [model code](https://github.com/alphagov/notifications-api/blob/541a765811dbb5592822f8ebcf87ddc87acebf16/app/models.py#L492-L493) and the type of the columns in the database before running this migration

Reply:
That's what I thought, that I already did it, too, but then those alter statements were generated automatically when I was creating the migration file, so I assumed it must have not happened. 

But since you checked and the type `Text` is there, I can drop those statements 👍🏼

Reply:
👍 The columns seem to be text in preview, staging and production, so maybe you just need to run those lines locally so your db matches.

Reply:
ahh, that makes sense, that will probably also solve the other automatic statements in my migrations. Sometimes I'm so slow 🤦🏼‍♀️. Thanks a lot!

Reply:
To be fair, I think most (all?) of the other automatic statements do need fixing!